src/js/api/api.js

```diff
@@ -50,13 +51,16 @@ define([
             if (_.isString(callback)) {
```

Comment:
We're still testing that this exception gets thrown. I like the idea of keeping *some* error or warning. I only added the isFunction check to pass a unit test where we're not passing in a function. Should we be as strict as possible or not at all?